tests/integration/test_store.py

```diff
@@ -470,6 +471,194 @@ def test_processing_quotas(
         assert event["logentry"]["formatted"] == f"otherkey{i}"
 
 
+@pytest.mark.parametrize("violating_bucket", [[4.0, 5.0], [4.0, 5.0, 6.0]])
+def test_rate_limit_metrics_buckets(
+    mini_sentry,
+    relay_with_processing,
+    metrics_consumer,
+    outcomes_consumer,
+    violating_bucket,
+):
+    """
+    param violating_bucket is parametrized so we cover both cases:
+        1. the quota is matched exactly
+        2. quota is exceeded by one
```

Comment:
The second parametrization will fail until https://github.com/getsentry/relay/pull/1521 gets merged.